### ISSUE TITLE
Registering

### ISSUE THREAD

**Issue by ViralBShah** [r9y9/OpenCV.jl#5]
Now that Cxx.jl is registered in METADATA, in preparation for 0.5, should we register OpenCV.jl too once the release is out?

Cc: @nkottary

**Comment by mprat** [comment]
It seems that before registering, the differences / similarities / merging of this package with https://github.com/maxruby/OpenCV.jl should be resolved.

**Comment by r9y9** [comment]
Yes, I'm happy with merging efforts. Our packages in JuliaOpenCV are not quite mature yet, but I would like to address #7 if we want to continue this work.

**Comment by ViralBShah** [comment]
cc @timholy

**Comment by timholy** [comment]
I'd *love* to have a good OpenCV wrapper; I was bummed that no one came forward for GSoC to do this.

As @mprat pointed out, would be great to compare notes with @maxruby. I'm about to hop on a plane, so I can't look carefully, but from a very brief inspection it seems that each has something to offer the other. More later, but feel free to start discussing without me.
